ERP/app/Http/Middleware/InventoryAccessOnly.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Closure;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+
+class InventoryAccessOnly
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Closure  $next
+     * @return mixed
+     */
+    public function handle(Request $request, Closure $next)
+    {
+        if(Auth::user()-> user_type == 1 || Auth::user()-> user_type == 2 || Auth::user()-> user_type == 3)
```

Comment:
Here, you are checking that if a user is of a certain type, then redirect.
What happens when we add another user type? You would need to change this IF statement and add that new user to the `if`

Can you change this to check that if the user is NOT of type IT or inventory, then redirect. This way, even if we add a new `user_type`, we don't need to modify this conditional

Reply:
you are right, I changed that